test_scripts/Polices/Policies_Security/002_ATF_P_Policies_Performance_Requirement.lua

```diff
@@ -0,0 +1,86 @@
+----  Name of requirement that is covered.
+----- [General] Policies performance requirement
+----  Mobile app is not blocked for other operations while waiting response from backend for updated Policy Table)
+
```

Comment:
@Aleksandriys update script with approved header and block names from our template